workflow/tests/util.rb

```diff
@@ -375,6 +375,10 @@ def _verify_outputs(rundir, hpxml_path, results, hpxml, unit_multiplier)
     if timestep > 15
       next if message.include?('Timestep: Requested number') && message.include?('is less than the suggested minimum')
     end
+    # Location doesn't match EPW station
+    if hpxml_path.include? 'base-location-detailed.xml'
+      next if message.include? 'Weather file location will be used rather than entered (IDF) Location object.'
```

Comment:
This warning is "back"? What made it go away?

Reply:
I originally used a location far from Denver, then decided to switch to Westminster. Now it's further away again.